datalake/src/main/java/com/sequenceiq/datalake/service/cloud/CloudResourceRetrieverService.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.datalake.service.cloud;
+
+import java.util.Optional;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+import com.sequenceiq.cloudbreak.cloud.service.ResourceRetriever;
+import com.sequenceiq.common.api.type.CommonStatus;
+import com.sequenceiq.common.api.type.ResourceType;
+
+@Component
+public class CloudResourceRetrieverService implements ResourceRetriever {
+
+    @Override
+    public Optional<CloudResource> findByResourceReferenceAndStatusAndType(String resourceReference, CommonStatus status, ResourceType resourceType) {
+        return Optional.empty();
```

Comment:
Looks like a missing implementation.

Reply:
Copied from redbeams service, this is also because of the added dependencies.